### DISCUSSION TITLE
Error:  name 'year' is not defined

### DISCUSSION THREAD

**Discussion by misterskye** [earthlab-education/earth-analytics-02-python-2024#5] [Q&A01 Urban Greenspace | Cloud Data]
I'm not sure what is going on, but for some reason I'm getting a _NameError: name 'year' is not defined_  when I attempt the NAIP search.  Not doing anything different then Elsa, but....

thanks,
sk

![image](https://github.com/earthlab-education/earth-analytics-02-python-2024/assets/47870472/495d9827-ece4-45e8-a0d6-05ac74b2fa6a)

**Comment by misterskye** [comment]
Doh!!!!!  Yeah, I figured it out

**Reply by edchanonline** [reply]
Haha, sounds like you didn't define `year` as a variable. Also, use `2021`. I didn't have any luck with `2023` data for example.

**Reply by misterskye** [reply]
Yeah, sometimes I don't quite think it thru, and then suddenly I realize what it wants.
